patroni/ctl.py

```diff
@@ -1168,8 +1168,9 @@ def restart(cluster_name: str, group: Optional[int], member_names: List[str],
 @option_citus_group
 @click.argument('member_names', nargs=-1)
 @option_force
+@click.option('--from-leader', is_flag=True, help='Set to sync wal from leader')
```

Comment:
I think the `help` section is wrong.
It is not about syncing WAL, but about getting basebackup.